guides/micronaut-cloud-database-aws/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "base": "micronaut-cloud-database-base",
+  "title": "Connect a Micronaut Data JDBC Application to an AWS MySQL Database",
```

Comment:
I think we need to add RDS to the title or description. We have to have the keyword for SEO.

Reply:
probably even the guide slug so that it appears in the url

Reply:
Done